modules/ossm-federation-overview.adoc

```diff
@@ -0,0 +1,17 @@
+////
+This module included in the following assemblies:
+- ossm-federation.adoc
+////
+
+
+[id="ossm-federation-overview_{context}"]
+= Federation overview
+
+[role="_abstract"]
```

Comment:
In the module before this, there's an improperly used italics for "minimal." It should be removed.

Reply:
I'm leaving the italics, as they point out a major difference between upstream and the product that we want to emphasize as a security feature.